cloudformation/membership-attribute-service.yaml

```diff
@@ -556,6 +572,31 @@ Resources:
           ReturnData: false
     DependsOn:
       - LoadBalancer
+  AttributesMismatchAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdMonitoring
+    Properties:
+      AlarmActions:
+        - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
```

Comment:
do we even need an alarm or just keep an eye on the metric every few days?  Or just send the alarm to one or two of us.  I don't see why we need to notify the whole group.

Reply:
Fair point, I've changed it to notify me only